modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_user_commands.adoc

```diff
@@ -395,6 +396,35 @@ All three arguments are mandatory and can not be empty. Example to set the email
 
 The email address of `carla` is updated to `foobar@foo.com`.
 
+== Move a Users Home Directory
```

Comment:
The command help refers to it as "folder". Should the heading here use the same?
```suggestion
== Move a Users Home Folder
```